src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileDownloadMenuItem.java

```diff
@@ -138,22 +138,19 @@ public FileHandle getFileHandle() {
 		return null;
 	}
 
+	
 	public String getDirectDownloadURL(FileEntity fileEntity, FileHandle fileHandle, String fileNameOverride) {
 		String externalUrl = null;
 		if (fileHandle instanceof ExternalFileHandle) {
 			externalUrl = ((ExternalFileHandle) fileHandle).getExternalURL();
 		}
 
 		String directDownloadURL = null;
-		if (externalUrl == null)
+		if (externalUrl != null && externalUrl.toLowerCase().startsWith(WebConstants.SFTP_PREFIX)) {
```

Comment:
Special case for external URL that starts with "sftp://", go to the proxy (just as it did before).